### ISSUE TITLE
Settings file are not beign loaded

### ISSUE THREAD

**Issue by heamaral** [aleab/toastify#93]
If I change some hotkey and save, when I restart toastify the settings returns to defaults.

**Comment by aleab** [comment]
Side-effect of poorly tested major changes to the hotkeys. I'm sorry! 😞
I'll look into this issue on Monday and release another hotfix afterwards.

**Comment by yogan** [comment]
I noticed this as well, happens with the v1.10.11 release. I switched back to v1.10.9 for that reason, where it still works fine (I don't need the song info feature, just having hotkeys is fine for me).

What I also noticed with v1.10.11: while I was trying to re-configure my hotkeys, I could set the checkbox for the Windows key. It would be set initially, but after saving or selecting another action, the checkbox was onset when I switched back to the first action. Not sure if this is related to the settings saving issue.
